src/Storage/Storage.Management/help/New-AzStorageContext.md

```diff
@@ -61,6 +61,18 @@ New-AzStorageContext [-StorageAccountName] <String> [-UseConnectedAccount] [-Pro
  -Environment <String> [<CommonParameters>]
 ```
 
+### AccountNameAndKeyServiceEndpoint
+```
+New-AzStorageContext [-StorageAccountName] <String> [-StorageAccountKey] <String> -BlobEndpoint <String>
+ [-FileEndpoint <String>] [-QueueEndpoint <String>] [-TableEndpoint <String>] [<CommonParameters>]
+```
+
+### SasTokenServiceEndpoint
+```
+New-AzStorageContext -SasToken <String> [-BlobEndpoint <String>] [-FileEndpoint <String>]
```

Comment:
Could you please submit a piece of design review at https://github.com/Azure/azure-powershell-cmdlet-review-pr?

I recommend the design of this cmdlet as following:
```
New-AzStorageContext -SasToken <String> -Endpoint <String> -Type <string>{Blob|File|Queue|Table}

```

Reply:
A design is already raised and pass review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1148

Since most time, a storage account will have all of B/T/Q/F endpoint input, the suggested design might not be proper.
Besides that, the cmdlet already has a parameter with name "-Endpoint" which means the common suffix of all endpoint.

Reply:
My mistake. I forget I reviewed this PR. Got it. Do you mean a storage account will have  B/T/Q/F endpoint input at the same time? If so, let's go ahead.

Reply:
Yes B/T/Q/F endpoints can be input at the same time.